subworkflows/nf-core/fasta_newick_epang_gappa/meta.yml

```diff
@@ -0,0 +1,82 @@
+name: "fasta_newick_epang_gappa"
+description: Run phylogenetic placement with a number of query sequences plus a reference alignment and phylogeny. Used in nf-core/phyloplace.
+keywords:
+  - phylogenetic placement
+  - phylogenetics
+  - alignment
+  - fasta
+  - newick
+modules:
+  - hmmer/hmmbuild
+  - hmmer/hmmalign
+  - hmmer/eslalimask
+  - hmmer/eslreformat
+  - epang/place
+  - epang/split
+  - gappa/examinegraft
+  - gappa/examineassign
+  - gappa/examineheattree
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - queryseqfile:
+      type: file
+      description: Fasta file with query sequences
+      pattern: "*.{faa,fna,fa,fasta,fa,faa.gz,fna.gz,fa.gz,fasta.gz,fa.gz}"
+  - refseqfile:
+      type: file
+      description: File with reference sequences; aligned unless an hmmfile is provided
+      pattern: "*.{faa,fna,fa,fasta,fa,phy,aln,alnfaa,alnfna,alnfa,mfa,faa.gz,fna.gz,fa.gz,fasta.gz,fa.gz,phy.gz,aln.gz,alnfaa.gz,alnfna.gz,alnfa.gz,mfa.gz}"
+  - refphylogeny:
+      type: file
+      description: Newick file with reference phylogenetic tree
+      pattern: "*.{newick,tree}"
+  - hmmfile:
+      type: file
+      description: HMM file to use for alignment; implies that refseqfile is not aligned.
+      pattern: "*.{hmm,HMM,hmm.gz,HMM.gz}"
+  - model:
+      type: string
+      description: Phylogenetic model to use in placement
```

Comment:
Maybe you have some examples here? In tests you have `LG` & `GTR+F+I+I+R3`, could you list here a few or link to available options or such?

Reply:
These strings come out of the programs that estimate the reference phylogeny, but I added a couple of examples to show uses what to look for.